### PR TITLE
WNP 103 for EU (Fixes #11905)

### LINKED ISSUE #11905: WNP 103 for EU
## Description

Copy doc: https://docs.google.com/document/d/1AYPQz2-4eBzzFrrqqCRbOqekNfKgJxl7MErMPrH2odU/edit?usp=sharing
Figma file: https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/EU_Firefox_WNP_21-22?node-id=1756%3A2040

---

#### Success Criteria
- [ ] FF 103 WNP for DE / FR / UK is in production by July 26

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP 103 for EU

## Issue / Bugzilla link

#11905

## Testing

- http://localhost:8000/en-GB/firefox/103.0/whatsnew/?geo=gb
- http://localhost:8000/en-US/firefox/103.0/whatsnew/?geo=gb
- http://localhost:8000/de/firefox/103.0/whatsnew/
- http://localhost:8000/fr/firefox/103.0/whatsnew/

### CODE REVIEW

**Comment by alexgibson** [comment]
This has been approved by stakeholders and is now ready for review

**Review by reemhamz — APPROVED** [review]
All looks good here, nice work! Just a few comments I thought necessary to bring up that we should get some clarification on/fix up. r+wc otherwise :) 🪂

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_103_eu') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main mzp-l-content mzp-t-content-md">
+  <h2 class="wnp-main-title">Explorez le Web en toute sécurité</h2>
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html:20]: Replay from Anastasia in Slack:

```
... in French it should be capitalised when used a a noun (not an adjective). 
We have the French rules here. Thanks for double checking!
```

https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  {% include 'firefox/whatsnew/includes/fx103/tcp.html' %}
+  <p class="wnp-main-tagline">
+    Firefox déploie son dispositif de protection de la vie privée le plus performant à ce jour :
+    la protection totale contre les cookies confine les cookies aux sites sur lesquels ils ont
+    été créés, ce qui empêche les traqueurs de les utiliser pour vous suivre lorsque vous naviguez
+    d’un site à l’autre. Firefox travaillera toujours à défendre votre vie privée et à innover
+    sans cesse pour votre confidentialité.
+  </p>
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html:28]: Good call out - will see what copy / design say here 👍

[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html:28]: They're fine with left aligned, updating!

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx103-fr.html:20]: TIL -- thanks for checking!

**Pull request merged**